README.md

```diff
@@ -133,6 +133,11 @@ alter table request_properties add column request_type character varying(255);
 alter table request_properties add column block_materials_request boolean;
 alter table request_properties add column he_slice_materials_request boolean;
 alter table request_properties add column others_materials_request character varying(255);
+update request_properties
+    set block_materials_request = true, he_slice_materials_request = true
+    where materials_request = true;
```

Comment:
I am afraid we need to do more something like
```sql
where process_instance_id in (
  select proc_inst_id_ from act_ru_variable where name_ = 'is_materials_request' and long_ = 1
);
```

Reply:
Because it is currently only stored as a request variable, so we need to consult the Activiti tables.

Reply:
ahh you're right! It was not one of the request property fields...